### PR TITLE
[AutoMM] Add the example of Conv-LoRA.

### PR DESCRIPTION
**Pull request by Harry-zzh**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |

**Review by zhiqiangdon — APPROVED** [review]
LGTM

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |

File: examples/automm/Conv-LoRA/run_semantic_segmentation.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+    # prepare dataframes
+    train_df = expand_path(pd.read_csv(os.path.join(dataset_dir, f"train.csv")), dataset_dir)
+    val_df = expand_path(pd.read_csv(os.path.join(dataset_dir, f"val.csv")), dataset_dir)
+
+    # get the validation metric
+    validation_metric = get_validation_metric(dataset_name)
+
+    hyperparameters = {}
+    hyperparameters.update(
```
Comment (zhiqiangdon) [inline examples/automm/Conv-LoRA/run_semantic_segmentation.py:65]: Need to specify `conv_lora` here?

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |
| filelock==3.13.2 | filelock==3.13.3 |
| filelock==3.13.2 | filelock==3.13.3 |

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |
| tensorrt==8.6.0 | tensorrt==8.6.1.post1 |
| tensorrt==8.6.0 | tensorrt==8.6.1.post1 |
| - | tensorrt-bindings==8.6.1 |
| - | tensorrt-libs==8.6.1 |

**Pull request merged**